stm32cube/stm32g0xx/README

```diff
@@ -36,12 +36,8 @@ License Link:
 
 Patch List:
 
-    *Extend public SetFlashLatency API to all families
-     The LL_SetFlashLatency public API is now defined for families
-     beyond G4, L4 and L5. The private function has been made public.
-    Impacted files:
-     drivers/include/stm32g0xx_ll_utils.h
-     drivers/src/stm32g0xx_ll_utils.c
-    ST Bug tracker ID: 76033
```

Comment:
Do you confirm thsi was solved in 1.4 ?

Reply:
Yes, I have compared the fix from the commit [stm32cube: extend public SetFlashLatency API to all families ](https://github.com/zephyrproject-rtos/hal_stm32/commit/6e2d3f951472597818416cfbfff9a601bc2e60e9#diff-13fd3a257e95248f2f8e93768fbe5e522bd48deb01970411bd7d921b2fee3a02) with the new release.
The existing fix renamed the UTILS_SetFlashLatency() function to LL_SetFlashLatency() and made it public.
In v1.4 this changes are included, and additionally a loop was added that waits until the new flash latency has been applied.